addons/partner_autocomplete/static/src/js/partner_autocomplete_core.js

```diff
@@ -36,7 +36,7 @@ export function usePartnerAutocomplete() {
         return checkVATNumber(sanitizeVAT(value));
     }
 
-    async function checkGSTNumber(value) {
+    async function isGSTNumber(value) {
```

Comment:
I just noticed this, but why is this function `async` ? It looks like it could be a regular function.